puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/rpm_packaging.groovy

```diff
@@ -24,7 +24,13 @@ pipeline {
             steps {
 
                 script {
-                    changed_packages = sh(returnStdout: true, script: "git diff origin/${ghprbTargetBranch} --name-only -- 'packages/**.spec' | xargs dirname | xargs -n1 basename |sort -u").trim()
+                    changed_packages = sh(returnStdout: true, script: "git diff origin/${ghprbTargetBranch} --name-only -- 'packages/**.spec'")
+
+                    if (changed_packages) {
+                        changed_packages = sh(returnStdout: true, script: "${changed_packages} | xargs dirname | xargs -n1 basename |sort -u").trim()
+                    } else {
+                        changed_packages = ''
```

Comment:
This splits the processing of changed_packages into two pieces to ensure that if nothing changes in `packages/` then it won't try to process that further.